src/UserService.Validation/User/CreateUserRequestValidator.cs

```diff
@@ -8,22 +8,34 @@ namespace LT.DigitalOffice.UserService.Validation.User
 {
     public class CreateUserRequestValidator : AbstractValidator<CreateUserRequest>, ICreateUserRequestValidator
     {
-        private static Regex NameRegex = new(@"\d");
+        private static Regex NumberRegex = new (@"\d");
+        private static Regex SpecialCharactersRegex = new (@"[$&+,:;=?@#|'<>.^*()%!-]");
+        private static Regex SpaceRegex = new (@"^[^@\s]*$");
+        private static Regex EmailRegex = new (@"^[^@\s]+@[^@\s]+\.[^@\s]*$");
+
         public CreateUserRequestValidator()
         {
             RuleFor(user => user.FirstName)
                 .NotEmpty()
-                .Must(x => !NameRegex.IsMatch(x))
+                .WithMessage("The field cannot be empty")
```

Comment:
write "The first name cannot be empty", not "field"

Reply:
done